### PR TITLE
Fix URL construction and refactor

### PR DESCRIPTION
**Pull request by LesterKim**
Fix the URL construction and refactor to make code DRY.

### CODE REVIEW

**Comment by LesterKim** [comment]
FYI @chilaraiSxt @nopslip @mrveera @v-ra-ins @austpryb

**Comment by LesterKim** [comment]
FYI @chilaraiSxt @nopslip @v-ra-ins @austpryb

File: authentication/authentication.go [hunk 1]
```diff
@@ -51,13 +48,12 @@ func GenerateAuthCode(userId, joinCode string) (authCode string) {
 	if err != nil {
 		log.Fatalln(err)
 	}
-	sb := string(body)
-	return sb
+	authenticationCode := string(body)
+	return authenticationCode
 }
```
Comment (v-ra-ins) [inline authentication/authentication.go:53]: Can we return without variable?

**Review by v-ra-ins — APPROVED** [review]
LGTM

**Review by chilaraiSxt — APPROVED** [review]
LGTM. Thanks!!

**Review by henrydaly — APPROVED** [review]

**Pull request merged**